src/bullsAndCows.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+/**
+ * Implement bullsAndCows function:
+ *
+ * Function takes 2 strings: generatedNumber (generated by computer) and
+ * enteredNumber (entered by player) and returns count of bulls and cows
+ * according to rules (see readme.md). Each parameter should be at most 4 digits
+ * long.
+ *
+ * Function returns result as array of numbers where result[0] is a count of
+ * bulls and result[1] is a count of cows.
```

Comment:
And it is a task requiring Arrays but not only strings and numbers

Reply:
I would prefer to have { bulls: 1, cows: 2 } object instead

Reply:
good point!